src/scss/layout/page/page.stories.ts

```diff
@@ -28,3 +34,35 @@ export const Page: Story = {
     ${Footer.render?.call({ ...args })}
   `,
 };
+
+export const Sidebar: Story = {
```

Comment:
Small thing but could we rename the two stories in this file to be something like `BasicLayout` and `SidebarLayout` to make it clearer for consumers?